vars/main.yml

```diff
@@ -0,0 +1,24 @@
+srsran:
+  docker:
+    container:
+      gnb_image: srsran-gnb:latest
+      ue_image: srsran-nr-ue:latest
```

Comment:
Should you update this? Such images do not exist, correct?

Reply:
done